MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -552,9 +919,18 @@ int main(int argc, char * argv[])
                 HRESULT hrMountVHD = MsixCoreLib::MountVHD(cli.GetMountImagePath(), cli.isMountReadOnly(), driveLetter);
                 if (FAILED(hrMountVHD))
                 {
+                    errorCode = msixmgr::GetErrorCodeFromHRESULT(hrMountVHD);
+                    errorDesc = L"Failed Mounting the VHD(X) file. HRESULT " + errorCode + L".";

```

Comment:
Same as above

Reply:
As discussed, all the Telemetry variables will have the same Error Message as the actual Console statements. The change for outputting the Error Message on Console from a variable will be done as part of Error Message Improvement Task that will be taken later.